com.obsproject.Studio.Plugin.SceneSwitcher.yaml

```diff
@@ -0,0 +1,45 @@
+id: com.obsproject.Studio.Plugin.SceneSwitcher
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/SceneSwitcher
+modules:
+  - name: opencv
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_LIST=core,imgproc,objdetect
+    sources:
+      - type: git
+        url: https://github.com/opencv/opencv.git
+        tag: 4.5.4
+        commit: 4223495e6cd67011f86b8ecd9be1fa105018f3b1
+  - name: advanced-scene-switcher
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_OUT_OF_TREE=1
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DLIBOBS_INCLUDE_DIR=/app/include/obs
+      - -DLIBOBS_LIB=/app/lib/libobs.so
+      - -DLIBOBS_FRONTEND_INCLUDE_DIR=/app/include/obs
+      - -DLIBOBS_FRONTEND_API_LIB=/app/lib/libobs-frontend-api.so
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: https://github.com/WarmUpTill/SceneSwitcher.git
+        tag: 1.16.4
+        commit: 5dfed8ac2ae27289d79f7900be460188aba747ab
```

Comment:
For extra convenience, you can integrate this with the [updates checker ](https://github.com/flathub/flatpak-external-data-checker)and let it manually update this manifest whenever you publish a release. For example:

```yaml
        x-checker-data:
          type: git
          tag-pattern: ^v([\d.]+)
```

Same for the `opencv` module